src/components/UploadPage/UploadPageComponents/VideoInput.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import "./VideoInput.scss"
+import type { UploadProps } from "antd";
+import { message, Upload } from "antd";
+
+const { Dragger } = Upload;
+
+const props: UploadProps = {
+  name: "file",
+  multiple: false,
+  action: "",
+  accept: "video/*",
```

Comment:
add one more `beforeUpload(file)` function and make it `return false` then it will remove the upload error as we will be posting content from frontend itself to distributed storage. Also then you will see the uploaded content would be listed and make sure it should be visible clearly. and please do this with all upload props.

Reply:
for reference you could refer [this](https://github.com/nlok5923/yeah-snap/blob/83bf4936f7b5d1bdc3c7b507b631c064dcdaa872/frontend/src/Pages/Dashboard/Create/Create.jsx#L30)

Reply:
Ok sir.. just implementing that...